### PR TITLE
Release 1.9.0

### PR DESCRIPTION
**Pull request by smurawski**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,33 +1,68 @@
 # Change Log
 
-## [1.8.0](https://github.com/test-kitchen/test-kitchen/tree/1.8.0) (2016-05-05)
```
Comment (mwrock) [inline CHANGELOG.md]: There may be a better way to do this but I would just copy what is generated for 1.9.0 and paste it onto the former changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,33 +1,68 @@
...
+[Full Changelog](https://github.com/test-kitchen/test-kitchen/compare/v1.8.0...1.9.0)
+
+**Implemented enhancements:**
+
+- Cannot set log level to "info" [\#529](https://github.com/test-kitchen/test-kitchen/issues/529)
+- Buffer errors until the end of an action [\#1034](https://github.com/test-kitchen/test-kitchen/pull/1034) ([smurawski](https://github.com/smurawski))
+
+**Fixed bugs:**
+
+- Use command\_prefix provided by Kitchen::Provisioner::Base in shell provisioner [\#1033](https://github.com/test-kitchen/test-kitchen/pull/1033) ([pstengel](https://github.com/pstengel))
```
Comment (mwrock) [inline CHANGELOG.md]: this looks like an enhancement

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,33 +1,68 @@
...
+
+**Merged pull requests:**
+
+- Fix grammar in common\_sandbox warning message [\#1031](https://github.com/test-kitchen/test-kitchen/pull/1031) ([emachnic](https://github.com/emachnic))
+- Update `chef\_omnbius\_url` default value [\#1028](https://github.com/test-kitchen/test-kitchen/pull/1028) ([schisamo](https://github.com/schisamo))
+- Empty string for the config setting for proxies did not really work [\#1027](https://github.com/test-kitchen/test-kitchen/pull/1027) ([smurawski](https://github.com/smurawski))
+- Added kitchen-docker [\#1025](https://github.com/test-kitchen/test-kitchen/pull/1025) ([jjasghar](https://github.com/jjasghar))
+- Add kitchen-azurerm to list of community-provided drivers [\#1024](https://github.com/test-kitchen/test-kitchen/pull/1024) ([stuartpreston](https://github.com/stuartpreston))
+- uploads: reuse connections+disable compression [\#1023](https://github.com/test-kitchen/test-kitchen/pull/1023) ([lamont-granquist](https://github.com/lamont-granquist))
+- ECOSYSTEM doc. [\#1015](https://github.com/test-kitchen/test-kitchen/pull/1015) ([jjasghar](https://github.com/jjasghar))
```
Comment (mwrock) [inline CHANGELOG.md]: I'd `no_changelog` the above edits by JJ and just leave this one and reword it a bit (also label as enhancement)

File: CHANGELOG.md (hunk 2 above)
Comment (smurawski) [inline CHANGELOG.md]: I'd say it was a bug - command prefix should have been supported there when it was implemented, but I'm not super invested in that opinion.

Comment (mwrock) [inline CHANGELOG.md]: tbh: neither am i. its fine.

**Comment by mwrock** [comment]
I'm :+1: but you might consider relabeling everything in `Merged pull requests` so thay fall under either bugs or enhancements.

**Comment by mwrock** [comment]
beautiful

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+## [1.9.0](https://github.com/test-kitchen/test-kitchen/tree/1.9.0) (2016-05-26)
+[Full Changelog](https://github.com/test-kitchen/test-kitchen/compare/v1.8.0...1.9.0)
+
+**Implemented enhancements:**
+
+- Cannot set log level to "info" [\#529](https://github.com/test-kitchen/test-kitchen/issues/529)
```
Comment (cheeseplus) [inline CHANGELOG.md]: We already shipped this one but closed the issue late

**Pull request merged**